.github/workflows/test.yml

```diff
@@ -6,23 +6,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        entry:
-          - { ruby: "2.7", grape-swagger: "1.6.1" }
-          - { ruby: "3.0", grape-swagger: "1.6.1" }
-          - { ruby: "3.1", grape-swagger: "1.6.1" }
-          - { ruby: "3.2", grape-swagger: "1.6.1" }
-          - { ruby: "3.2", grape-swagger: "2.0.2" }
-          - { ruby: "jruby-9.4.6", grape-swagger: "1.6.1" }
-          - { ruby: "jruby-9.4.6", grape-swagger: "2.0.2" }
+        ruby: ["3.1", "3.2", "jruby-9.4.6"]
+        rails: ["6.1.7", "7.2.1"]
+        grape-swagger: ["1.6.1", "2.0.2"]
```

Comment:
The versions above this line will be tested in all possible permutations.

Reply:
I usually find that rather wasteful as the matrix explodes with 3 different versions.

Reply:
I agree that this quickly leads to larger amounts of test runs if we're not careful.

The main question here is: which combinations do you want to officially support then? Are you generally OK with the resulting combinations outlined in the PR description, keeping in mind that following the same approach will add 4 more permutations for Ruby 3.3?

Or should I instead remove some of those permutations, for example drop official support for grape-swagger 1.x after a certain Ruby version?